modules/grpc-java/1.68.1/MODULE.bazel

```diff
@@ -0,0 +1,208 @@
+module(
+    name = "grpc-java",
+    compatibility_level = 0,
+    repo_name = "io_grpc_grpc_java",
+    version = "1.68.1",  # CURRENT_GRPC_VERSION
+)
+
+# GRPC_DEPS_START
+IO_GRPC_GRPC_JAVA_ARTIFACTS = [
+    "com.google.android:annotations:4.1.1.4",
+    "com.google.api.grpc:proto-google-common-protos:2.29.0",
+    "com.google.auth:google-auth-library-credentials:1.23.0",
+    "com.google.auth:google-auth-library-oauth2-http:1.23.0",
+    "com.google.auto.value:auto-value-annotations:1.11.0",
+    "com.google.auto.value:auto-value:1.11.0",
+    "com.google.code.findbugs:jsr305:3.0.2",
+    "com.google.code.gson:gson:2.11.0",
+    "com.google.errorprone:error_prone_annotations:2.28.0",
+    "com.google.guava:failureaccess:1.0.1",
+    "com.google.guava:guava:33.2.1-android",
+    "com.google.j2objc:j2objc-annotations:3.0.0",
+    "com.google.re2j:re2j:1.7",
+    "com.google.truth:truth:1.4.2",
```

Comment:
Pretty sure that's preexisting, but leaking test deps into the shared `maven` namespace isn't ideal. Not sure what the Bzlmod situation is upstream though.

Reply:
We patch these out in our own use of grpc-java